src/Assimp/Silk.NET.Assimp/Structs/AABB.gen.cs

```diff
@@ -21,12 +22,24 @@ public unsafe partial struct AABB
     {
         public AABB
         (
-            System.Numerics.Vector3 mMin = default,
-            System.Numerics.Vector3 mMax = default
+            System.Numerics.Vector3? mMin = null,
+            System.Numerics.Vector3? mMax = null
         )
         {
-            MMin = mMin;
-            MMax = mMax;
+            fixed (AABB* @this = &this)
```

Comment:
?? why is this better then `: this()` (I know generated code, but this just seems to be unnecessarily unsafe)

Reply:
Will that work on structs? The idea here is to force the runtime to initialise all fields to default so that we don't have to do it, which caused problems in the past.

Reply:
Now uses : this()